src/Components/Goods/Goods.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type Props = {
+  goods: Array<string>;
+};
+
+export const Goods: React.FC<Props> = ({ goods }) => (
+  <ul>
+    <ul>
+      {goods.map((good: string, index: number) => (
+        <li key={index} data-cy="Good">
```

Comment:
```suggestion
      {goods.map((good: string) => (
        <li key={good} data-cy="Good">
```

The name will be preferable, since it is unique in this array, it is better not to use the index as a key since the array can change, after which certain operations with the elements of the array may not work correctly, so it is worth using one of the element values, in our case the name of the product itself will be unique.